recipes/pastrami/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+#chmod +x $SRC_DIR/pastrami.py
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
[Calling `setup.py` directly like this is deprecated.](https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html)

Reply:
Thanks! I was not aware. Done!